config/api.env.example

```diff
@@ -88,14 +90,32 @@
 #     	open telemetry HTTP context pass and trace (default "true")
 #   WORKER_QUEUE string
 #     	job worker implementation (memory, redis, sqs, postgres) (default "memory")
-#   WORKER_CONCURRENCY int
-#     	number of goroutines handling jobs (default "50")
-#   WORKER_HEARTBEAT int64
-#     	heartbeat interval (time interval syntax) (default "30s")
-#   WORKER_MAX_BEATS int
-#     	maximum amount of heartbeats allowed (default "10")
+#   WORKER_POLL_INTERVAL int64
+#     	polling interval (network timeout) (default "5s")
+#   WORKER_MAX_THREADS int
+#     	maximum allowed amount of polling goroutines (CPUs + 1) (default "64")
+#   UNLEASH_ENABLED bool
+#     	unleash service (feature flags) (default "false")
+#   UNLEASH_ENVIRONMENT string
+#     	unleash environment (default "")
+#   UNLEASH_PREFIX string
+#     	unleash flag prefix (default "provisioning")
+#   UNLEASH_URL string
+#     	unleash service URL (default "http://localhost:4242")
+#   UNLEASH_TOKEN string
+#     	unleash service client access token (default "")
+#   KAFKA_ENABLED bool
+#     	kafka service enabled (default "false")
+#   KAFKA_BROKERS slice
+#     	kafka hostname:port list of brokers (default "localhost:9092")
+#   KAFKA_AUTH_TYPE string
+#     	kafka authentication type (mtls, sasl or empty) (default "")
+#   KAFKA_CA_CERT string
+#     	kafka TLS CA certificate path (default "")
```

Comment:
We should check these by GH actions... the refresh was not executed for some time.

Reply:
Agreed. :-)